README.md

```diff
@@ -1,20 +1,22 @@
 # cfgov-refresh
 
-The in-progress redesign of the [consumerfinance.gov](http://consumerfinance.gov) website.
-This project includes the front-end assets and build tools,
+The in-progress redesign of the [consumerfinance.gov](http://consumerfinance.gov)
+website. This project includes the front-end assets and build tools,
```

Comment:
For maximally-semantic line breaks, please have each sentence begin on a new line.